pkgs/data/fonts/iosevka/package.json

```diff
@@ -1,26 +1,29 @@
 {
 	"name": "iosevka-build-deps",
-	"version": "2.3.3",
+	"version": "3.0.1",
```

Comment:
Should we have 

```json
"engines": {
	"node": ">=12.16.0"}
```

here?

Reply:
I don't see why not!

Reply:
I think I misunderstood the goal of this file. IIUC it's not there to be evaluated by NPM to build the package (upstream's `package.json` is used for that) - this is just here to know which packages need to be added to the `nodePackages` package set.

Sorry for the confusion, I think the addition I suggested is actually not needed.